.github/workflows/presubmit.yaml

```diff
@@ -17,6 +17,14 @@ jobs:
     - uses: ./.github/actions/install-deps
       with:
         k8sVersion: ${{ matrix.k8sVersion }}
+    - name: Install kube-apiserver (k8s v1.25 only)
+      # Install kube-apiserver v1.25.16
+      if: ${{ matrix.k8sVersion == '1.25.x' }}
+      run: |
+          tempout=$(mktemp)
+          wget -P "$tempout" dl.k8s.io/v1.25.16/bin/linux/amd64/kube-apiserver
```

Comment:
Can we also make sure we download all of the associated binaries. There should be three that are downloaded with `setup-envtest` (`kubectl`, `etcd`, and `kube-apiserver`)

Reply:
Is there a need to grab the other binaries or is it just for consistency? My thought was to leave as much as possible envtests default distribution. (plus etcd isn't distributed or versioned with the other binaries)

Reply:
Discussed offline, now includes `kubectl`. I haven't included `etcd` since setup-envtest already installs the correct version.